src/app/app-extras.module.ts

```diff
@@ -2,35 +2,42 @@ import {
   NgModule
 } from '@angular/core';
 
+import {
+  SkyTilesModule
+} from '@skyux/tiles';
+
 import {
   SkyConfirmModule,
   SkyModalModule,
   SkyModalService
 } from './public';
 
-import {
-  SkyTilesModule
-} from '@skyux/tiles';
-
 import {
   ModalDemoComponent
 } from './visual/modal/modal-demo.component';
+
 import {
   ModalContentDemoComponent
 } from './visual/modal/modal-content-demo.component';
+
 import {
   ModalFullPageDemoComponent
 } from './visual/modal/modal-fullpage-demo.component';
+
 import {
   ModalLargeDemoComponent
 } from './visual/modal/modal-large-demo.component';
+
 import {
   ModalTiledDemoComponent
 } from './visual/modal/modal-tiled-demo.component';
+
 import {
   ModalContentAutofocusComponent
 } from './visual/modal/modal-content-autofocus.component';
 
+require('style-loader!./visual.scss');
```

Comment:
What's this all about?

Reply:
This will hide the red SKY UX Host status bar in the screenshots. We should actually add this to all other modules, too.